knowledge-base/kb-gridview-howto-hide-aggregate-results-from-groupheaderrow.md

```diff
@@ -0,0 +1,59 @@
+---
+title: How to Hide Some Aggregate Results From GroupHeaderRow
+description: Filter Aggregate Results Displayed in GroupHeaderRow
+type: how-to
+page_title: Remove Aggregate Results when RadGridView is grouped
+slug: kb-gridview-howto-hide-aggregate-results-from-groupheaderrow
+position: 0
+ticketid: 1404936
+tags: gridview, groupheaderrow, aggregateresults, filter, aggregateresultslist, hide
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>RadGridView for WPF and Silverlight</td>
+	</tr>
+</table>
+
+## Description
+
+By default, when the RadGridView is grouped, the GroupHeaderRow shows the aggregate results of the [AggregateFunctions]({%slug gridview-columns-aggregate-functions%}) defined for all columns. This is the default behavior of the __AggregateResultsList__, which resides in the GroupHeaderRow ControlTemplate. This article will demonstrate how this behavior can be changed. 
```

Comment:
Usually, the description should be more like a customer asking than documentation style. You can consider to revamp this paragraph.

- Maybe you can use something like - "How to hide the aggregate results from the GroupHeaderRow when RadGridView is grouped, and aggregate functions are used."